src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -342,3 +362,6 @@ function edmfx_sgs_diffusive_flux_tendency!(
 
     return nothing
 end
+
+dissipation(ρatke⁰, tke⁰, l, c_d, dt) =
+    tke⁰ >= 0 ? c_d * ρatke⁰ * sqrt(tke⁰) / max(l, 1) : ρatke⁰ / dt
```

Comment:
Just note that this is assuming ρatke = ρa * tke, which is not necessarily true because of `divided_by_ρa`, but I think it's ok.